dotnet-client-libraries/WireMock.NET/WireMockNet/Planet.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace WireMockNet
+{
+    public class Planet
```

Comment:
This is a good candidate for a record, which will give you the Equality check for free and will then help to make the assertions in the test cleaner

Reply:
Resolved in a separate PR https://github.com/CodeMazeBlog/CodeMazeGuides/pull/1182